plugins/chat/assets/javascripts/discourse/models/chat-message-reaction.js

```diff
@@ -0,0 +1,33 @@
+import { tracked } from "@glimmer/tracking";
+import User from "discourse/models/user";
+import { TrackedArray } from "@ember-compat/tracked-built-ins";
+
+export default class ChatMessageReaction {
+  static create(args = {}) {
```

Comment:
Same wondering about why this is needed when we can just do `new ChatMessageReaction`

Reply:
see https://github.com/discourse/discourse/pull/20397#discussion_r1122750585